source/simulator/parameters.cc

```diff
@@ -160,29 +162,46 @@ namespace aspect
                        "This parameter indicates whether the simulator should also use "
                        "heat conduction in determining the length of each time step.");
 
-    prm.declare_entry ("Nonlinear solver scheme", "IMPES",
-                       Patterns::Selection ("IMPES|iterated IMPES|iterated Stokes|Newton Stokes|Stokes only|Advection only"),
+    prm.declare_entry ("Nonlinear solver scheme", "single Advection, single Stokes",
+                       Patterns::Selection ("single Advection, single Stokes|iterated Advection and Stokes|single Advection, iterated Stokes|no Advection, iterated Stokes|iterated Advection and Newton Stokes|single Advection, no Stokes|IMPES|iterated IMPES|iterated Stokes|Newton Stokes|Stokes only|Advection only"),
```

Comment:
You can make this easier to read, if you specify the selection string above as follows:
```
const std::string allowed_solver_schemes = "single Advection, single Stokes|"
                                                       "iterated Advection and Stokes|"
 .... 
                                                       ;

and then use `Patterns::Selection(allowed_solver_schemes)`